web-ng/src/app/components/drawing-tools/drawing-tools.component.ts

```diff
@@ -81,11 +85,38 @@ export class DrawingToolsComponent implements OnInit, OnDestroy {
         this.drawingToolsService.setSelectedLayerId(this.selectedLayerId);
       }),
       map(project =>
-        List(project.layers.valueSeq().toArray()).sortBy(l => l.index)
+        this.addableLayers(
+          project,
+          authService.getCurrentUser().email,
+          'points'
+        )
       )
     );
   }
 
+  private addableLayers(
```

Comment:
Can we move this into the `ProjectService`, as in #621, and recast as`canUserAddPointsToLayer(Layer)`? Then the `pipe` can `filter` instead of `map`,  the method name will be more self documenting, and won't rely on the string representation. Wdyt?

Reply:
Sounds good, changed.